production/loki-mixin/dashboards/loki-writes.libsonnet

```
@@ -54,13 +54,13 @@ local utils = import 'mixin-utils/utils.libsonnet';
                           $.row('Distributor')
                           .addPanel(
                             $.panel('QPS') +
-                            $.qpsPanel('loki_request_duration_seconds_count{%s}' % std.rstripChars(dashboards['loki-writes.json'].distributorSelector, ','))
+                            $.qpsPanel('loki_request_duration_seconds_count{%s route=~"api_prom_push|loki_api_v1_push|/httpgrpc.HTTP/Handle"}' % std.rstripChars(dashboards['loki-writes.json'].distributorSelector, ','))
```

It should add a common since it is stripping it off?
```suggestion
                            $.qpsPanel('loki_request_duration_seconds_count{%s, route=~"api_prom_push|loki_api_v1_push|/httpgrpc.HTTP/Handle"}' % std.rstripChars(dashboards['loki-writes.json'].distributorSelector, ','))
```